### PR TITLE
Update URI

### PR DESCRIPTION
**Pull request by safareli**

### CODE REVIEW

**Review by garyb — APPROVED** [review]
I was going to suggest we just pass options with the URI to `tileLayer` / `setURI`, but I see that's perhaps a bit more of a problem for `ConvertDict`.

File: src/Leaflet/Core/Types.purs [hunk 1]
```diff
@@ -132,13 +154,13 @@ type PopupConf =
 
 type IconConf =
   InteractiveLayerConf
-  ( iconUrl ∷ URIRef
-  , iconRetinaUrl ∷ URIRef
+  ( iconUrl ∷ LeafURIRef
```
Comment (cryogenian) [inline src/Leaflet/Core/Types.purs:157]: `iconUrl`, `iconRetinaUrl`, `shadowUrl` and `shadowRetinaUrl` are common uris btw, they don't have any special meanings for `{{x|y|z}}`

Comment (safareli) [reply src/Leaflet/Core/Types.purs:157]: I don't understand what you mean, can you explain ?

Comment (cryogenian) [reply src/Leaflet/Core/Types.purs:157]: `setURI` function sets special uri that could have `{` and `}` inside. `iconUrl` and friends couldn't they are just old plain URI w/o any fancy template strings inside.

**Pull request merged**